Framework/Core/include/Framework/AnalysisHelpers.h

```diff
@@ -11,41 +11,91 @@
 #ifndef o2_framework_AnalysisHelpers_H_DEFINED
 #define o2_framework_AnalysisHelpers_H_DEFINED
 
-#include "Framework/DataAllocator.h"
-#include "Framework/Traits.h"
-#include "Framework/TableBuilder.h"
 #include "Framework/ASoA.h"
-#include "Framework/OutputSpec.h"
-#include "Framework/OutputRef.h"
+#include "Framework/DataAllocator.h"
+#include "Framework/ExpressionHelpers.h"
+#include "Framework/IndexBuilderHelpers.h"
 #include "Framework/InputSpec.h"
-#include "Framework/OutputObjHeader.h"
-#include "Framework/StringHelpers.h"
 #include "Framework/Output.h"
-#include "Framework/IndexBuilderHelpers.h"
+#include "Framework/OutputObjHeader.h"
+#include "Framework/OutputRef.h"
+#include "Framework/OutputSpec.h"
 #include "Framework/Plugins.h"
-#include "Framework/ExpressionHelpers.h"
+#include "Framework/StringHelpers.h"
+#include "Framework/TableBuilder.h"
+#include "Framework/Traits.h"
 
 #include <string>
+namespace o2::soa
```

Comment:
Given TableRef is a new class, we can simply add it and introduce these helpers, no?

Reply:
It is independent and can be added separately. Should I put it into a separate header then?

Reply:
As you prefer. I think we can introduce any internal new classes / structs / concepts as long as we do not use them and there is no side effects or if they are merely a cleanup (e.g. `soa::is_table` as you have elsewhere).